sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/AlterTableAddPartitionSuite.scala

```diff
@@ -46,6 +47,24 @@ trait AlterTableAddPartitionSuiteBase extends command.AlterTableAddPartitionSuit
     }
   }
 
+  private def copyPartition(t: String, from: String, to: String): String = {
```

Comment:
ditto: rename `t` (and perhaps `from` and `to` as well) to something more meaningful

Reply:
Not sure about `from` and `to`. What do you propose here?